### ISSUE TITLE
Click on table prefix doesn't work anymore

### ISSUE THREAD

**Issue by Z-28** [phpmyadmin/phpmyadmin#17947]
### Describe the bug

Phpmyadmin groups tables with the same prefix in the left menu. Until recently I could click on the prefix to get a list of all tables with that prefix in the main window. That doesn't work anymore. If I click on the prefix nothing happens. The prefix isn't even a link anymore. Note that I'm talking about clicking the prefix itself, not the plus sign to the left of the prefix.

### To Reproduce

Create a few tables with the same prefix so that they becomes grouped in the left menu. Then click on the name of the prefix (not the plus sign to the left

### Expected behavior

You should get a list of all tables with the specific prefix in the main window

### Screenshots

If applicable, add screenshots to help explain the bug.
![phpmyadmin_bug](https://user-images.githubusercontent.com/93569439/205434288-afaf0610-1b64-4adb-b5ba-b671e6880d5c.png)

### Server configuration

- Operating system: Centos 7
- Web server: Apache
- Database version: 10.10.2-MariaDB - MariaDB Server
- PHP version: 8.1.13
- phpMyAdmin version: 5.2.0

### Client configuration

- Browser: Chromium
- Operating system: Linux

### Additional context

Add any other context about the bug here.

**Comment by williamdes** [comment]
I agree, I also was searching for this link recently

**Comment by williamdes** [comment]
I tried to debug this but it has some deep things to fix

**Comment by eddi13** [comment]
I would like to add a little more:
For group link, group separator (NavigationTreeTableSeparator) is ignored.
If the groups have the same beginning, i.e. "group__" and "grouptmp__", and the group "group__" is clicked, the tables of both groups are displayed.
You can see this on the screenshot.
![grouptest](https://github.com/phpmyadmin/phpmyadmin/assets/2987957/3b6c85d7-ea0d-45ca-adf9-a4cc8b8e57d4)

**Comment by williamdes** [comment]
Thank you @eddi13

**Comment by deg026** [comment]
i found fast solution to fix this issue, because cannot work anymore without this feature...
fast fix of this bug:

libraries/classes/Navigation/NavigationTree.php line 1103
change:
`if (! $node->isGroup) {`
to:
`if (! $node->isGroup || $node->links['text']['route']) {`

templates/navigation/tree/node.twig line 22
change block:
```twig
    {% if node.isGroup %}
      <div class="block second{{ has_second_icon ? ' double' }}">
        <u>{{ get_image(node.icon['image'], node.icon['title']) }}</u>
      </div>
      &nbsp; {{ node.name }}
    {% else %}
```
to block:
```twig
    {% if node.isGroup and text_link.route %}
      <div class="block second{{ has_second_icon ? ' double' }}">
        <u>{{ get_image(node.icon['image'], node.icon['title']) }}</u>
      </div>
      &nbsp;<a class="hover_show_full" href="{{ url(text_link.route, text_link.params) }}">{{ node.name }}</a>
    {% elseif node.isGroup %}
      <div class="block second{{ has_second_icon ? ' double' }}">
        <u>{{ get_image(node.icon['image'], node.icon['title']) }}</u>
      </div>
      &nbsp; {{ node.name }}
    {% else %}

```

done, powerful feature is works fine again!

**Comment by kamil-tekiela** [comment]
@deg026 You can submit a Pull Request

**Comment by deg026** [comment]
> [@deg026](https://github.com/deg026) You can submit a Pull Request

sorry, dont have configured git for phpmyadmin on my PC, can i do it somehow without it?

**Comment by kamil-tekiela** [comment]
Yes you can do it right here in the browser. GitHub allows you to create simple PRs with cloning the repo locally.

**Comment by deg026** [comment]
#19798
